build/images/inspect.sh

```diff
@@ -12,36 +12,65 @@ function usage() {
 
 function skopeo-inspect() {
     local img="docker://$1"
+    local creds=""
+    [[ "${1}" == artifactory.algol60.net/* ]] && creds="${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}"
     # echo >&2 "+ skopeo inspect $img"
     docker run --rm "$SKOPEO_IMAGE" \
         --command-timeout 60s \
         --override-os linux \
         --override-arch amd64 \
         inspect \
         --retry-times 5 \
-        --creds "${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}" \
+        ${creds:+--creds "${creds}"} \
```

Comment:
Providing artifactory credentials causes skopeo to fail inspection of `quay.io/skopeo:1.4.1`. Made inclusion of `--creds` conditional.